test/test_sbml_conversion.py

```diff
@@ -92,10 +92,14 @@ def parameter_estimation(
                               x_fixed_vals=[pars[idx] for idx in fixed_pars]
                               )
 
-    optimize_options = pypesto.OptimizeOptions(allow_failed_starts=False)
+    optimize_options = pypesto.OptimizeOptions(
+        allow_failed_starts=False,
+        startpoint_resample=True,
+    )
 
     results = pypesto.minimize(
-        problem, optimizer, n_starts, options=optimize_options)
+        problem, optimizer, n_starts, options=optimize_options,
```

Comment:
The comma here looks weird... Is it really supposed to be?

Reply:
that's totally allowed in python

Reply:
I would actually argue that it's good style because it makes things like reordering arguments and extension of the arguments a little bit easier.